lib/livebook/notebook/cell/input.ex

```diff
@@ -67,6 +67,15 @@ defmodule Livebook.Notebook.Cell.Input do
     end
   end
 
+  defp validate_value(value, :color) do
+    color_regex = ~r/#[0-9a-f]{6}/
+
+    case String.match?(value, color_regex) do
```

Comment:
Note we already validate colors somewhere else in this app at User.hex_color_valid?, perhaps we should unify them? Also, note a-f can also be upcased.

Reply:
Definitely makes sense! Do you have any thoughts on where it'd make sense to put a shared function? My first instinct is a new module `Livebook.Utils.ValidationUtils` or something like that, but doesn't feel great. On the casing, I was going by [this MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color#value) that says it's always stored lower-case by the input element, but agreed that we can validate independent of case. 

I think it makes most sense to move the existing `hex_color_valid?`somewhere shared and use it as-is. Just not sure where the best place for that is!

Reply:
I think @josevalim replied in https://github.com/elixir-nx/livebook/issues/409#issuecomment-869210709 ^^

Reply:
Ugh, my bad. I deleted it there, here is the comment:

> I would add to Utils itself, as there is already a valid_url? in there. :)

Reply:
Thanks! I've added `valid_hex_color?` to `Livebook.Utils` :)